src/pages/management/configuration-api/index.mdx

```diff
@@ -2531,6 +2531,18 @@ Tag can be used to mark a chat's thread only if the thread and the tag are assig
 
 </Section>
 
+## Sample App
```

Comment:
How about we move this section below Methods? So that "Methods" is always the first section in each category

Reply:
Yup, I agree. Looks better when I check on the localhost😊